packages/nx-go/src/go-package-graph/index.ts

```diff
@@ -12,6 +12,12 @@ export const processProjectGraph = (graph: ProjectGraph, context: ProjectGraphPr
     projectRootLookupMap.set(graph.nodes[projectName].data.root, projectName)
   }
 
+  const fileImportsInfoJsonStr = execSync(`go run ${__dirname}/get-go-imports.go .`, {
+    encoding: 'utf-8',
+    cwd: workspaceRootPath,
+  })
+  const fileImportsMap = JSON.parse(fileImportsInfoJsonStr)
```

Comment:
**suggestion:** Create a type defining the expected structure so typescript can help with type checking.

```ts
// Add at the bottom of the file
interface GoCodeImports {
 [key: string]: string[]
}
...

const fileImportsMap = JSON.parse(fileImportsInfoJsonStr) as GoCodeImports
```

Also rename `fileImportsMap` to something a little more descriptive, maybe `goFileImportMap`?